commands/project/dev/unifiedFlow.ts

```diff
@@ -25,27 +26,20 @@ import SpinniesManager from '../../../lib/ui/SpinniesManager';
 import LocalDevProcess from '../../../lib/projects/localDev/LocalDevProcess';
 import LocalDevWatcher from '../../../lib/projects/localDev/LocalDevWatcher';
 import { handleExit, handleKeypress } from '../../../lib/process';
-import {
-  isAppDeveloperAccount,
-  isStandardAccount,
-} from '../../../lib/accountTypes';
-import { uiCommandReference } from '../../../lib/ui';
-import { i18n } from '../../../lib/lang';
+import { uiLogger } from '../../../lib/ui/logger';
+import { commands } from '../../../lang/en';
+import { isDeveloperTestAccount, isSandbox } from '../../../lib/accountTypes';
 // import LocalDevWebsocketServer from '../../../lib/projects/localDev/LocalDevWebsocketServer';
 
 export async function unifiedProjectDevFlow(
   args: ArgumentsCamelCase<ProjectDevArgs>,
-  accountConfig: CLIAccount,
+  initialTargetProjectAccountId: number,
+  initialTargetTestingAccountId: number,
```

Comment:
Nit: This is a pretty high number of args, might be better to have it take an object as an arg